src/views/OrderDetail.vue

```diff
@@ -0,0 +1,217 @@
+            <ion-checkbox v-else @click="checkProducts(item.shopifyProductSKU)"/>
+            <ion-button fill="clear" color="medium">
+              <ion-icon slot="icon-only" :icon="ellipsisVerticalOutline" />
+            </ion-button>
+          </div>
+        </div>
+      </div>
+    </ion-content>   
+  </ion-page>
+</template>   
+<script lang="ts">
+import Image from '@/components/Image.vue';
+
+import { defineComponent } from 'vue';
+import { mapGetters, useStore } from "vuex";
+import { useRouter } from 'vue-router';
+import { DateTime } from 'luxon';
+import { IonPage, IonHeader, IonToolbar, IonBackButton, IonTitle, IonContent, IonSearchbar, IonItem, IonThumbnail, IonLabel, IonInput, IonChip, IonIcon, IonButton, IonCheckbox, IonSelect, IonSelectOption } from '@ionic/vue'
+import { ellipsisVerticalOutline, sendOutline, checkboxOutline, arrowUndoOutline } from 'ionicons/icons'
+export default defineComponent({
+    components: {
+      Image,
+      IonPage,
+      IonHeader,
+      IonToolbar,
+      IonBackButton,
+      IonTitle,
+      IonContent,
+      IonSearchbar,
+      IonItem,
+      IonThumbnail,
+      IonLabel,
+      IonInput,
+      IonChip,
+      IonIcon,
+      IonButton,
+      IonCheckbox,
+      IonSelect,
+      IonSelectOption
+    },
+    computed: {
+      ...mapGetters({
+        ordersList: 'order/getOrdersList',
+        getProduct: 'product/getProduct',
+      }),
+    },
+    data() {
+      return {
+        numberOfDays: 0,
+        numberOfPieces: 0,
+        catalog: "",
+        listOfCheckedProducts: [] as any,
+      }
+    },
+    methods: {
+      checkProducts(productSku: string) {
+        if (this.listOfCheckedProducts.includes(productSku)) {
+          this.listOfCheckedProducts.splice(this.listOfCheckedProducts.indexOf(productSku, 1));
+        }
+        else {
+          this.listOfCheckedProducts.push(productSku);
+        }
+      },
+      apply() {
+        this.ordersList.items.map((item: any) => {
+          if (this.listOfCheckedProducts.includes(item.shopifyProductSKU)) {
+            item.quantityOrdered -= this.numberOfPieces;
+            item.arrivalDate = DateTime.fromFormat(item.arrivalDate, "D").plus({ days: this.numberOfDays }).toFormat('MM/dd/yyyy');
+            if (this.catalog == "Preorder") item.isNewProduct = true
+            else item.isNewProduct = false
+          }
+        })
+        this.store.commit('order/order/ITEMS_UPDATED', this.ordersList.items);
+      },
+      getGroupList (items: any) {
+        return Array.from(new Set(items.map((ele: any) => ele.parentProductId)))
+      },
+      getGroupItems(parentProductId: any, items: any) {
+        return items.filter((item: any) => item.parentProductId == parentProductId)
+      },
+      checkAllProducts() {
+        this.ordersList.items.forEach((item: any) => {
+          this.checkProducts(item.shopifyProductSKU);
```

Comment:
If one of them is already selected this will unselect it, but it should select all